ajax/libs/angular-bootstrap-grid-tree/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "angular-bootstrap-grid-tree",
```

Comment:
Let us use "tree-grid-directive" which is the same as repo name. Also do not forget to revise folder name and PR title.